Makefile

```diff
@@ -40,9 +40,9 @@ generateExtensionPackages:
 	go run package-manager/cmd/populator
 
 test: test-setup
-	staticcheck ./internal/app/...
-	go vet ./internal/app/...
-	go test -v ./internal/app/... -coverprofile=coverage.out -covermode count
+	staticcheck ./internal/app/... ./cmd/populator/...
```

Comment:
If you want to add the populator to the PR test, please make sure the quality gate check passes.

Reply:
I removed it. Making the gate check passing involves creating unit tests to cover 75% of the package code which I think is a bite out of the scope of this ticket. We could create a new one to increase coverage for the `populator` package. cc: @jnewton03